src/modules/FioAddress/util.ts

```diff
@@ -120,6 +120,28 @@ export const setFioExpiredCheckToDisklet = async (lastChecks: { [fioName: string
   }
 }
 
+export const fioMakeSpend = async (fioWallet: EdgeCurrencyWallet, actionName: string, params: any): Promise<EdgeTransaction> => {
```

Comment:
I would do `params: unknown` instead of `any`. This is because `unknown` forces you to clean before touching, so it's safer.